server/app.js

```diff
@@ -0,0 +1,17 @@
+if (process.env.NODE_ENV == 'development') {
+  require('dotenv').config()
+}
+
+const express = require('express')
+const app = express()
+const PORT = 3000
```

Comment:
kasih aja di env yakkk
```suggestion
const PORT = process.env.PORT || 3000
```